imports/plugins/core/accounts/client/components/email/updateEmail.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Meteor } from "meteor/meteor";
+import { i18next } from "/client/api";
+import { Components } from "@reactioncommerce/reaction-components";
+
+class UpdateEmail extends Component {
+  static propTypes = {
+    email: PropTypes.string,
+    uniqueId: PropTypes.string
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: props.email,
+      newEmail: false,
+      showSpinner: false
+    };
+
+    this.handleFieldChange = this.handleFieldChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
These bindings aren't needed because you defined those as arrow functions (accomplishes the same thing).